### PR TITLE
DEV-3997 make the mobile branch audits beautiful

### PR DESCRIPTION
**Pull request by nprimo**

### CODE REVIEW

File: subjects/mobile-dev/01-maps/audit/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #### Functional
 
-#### In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app).
+> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
```
Comment (mikysett) [inline subjects/mobile-dev/01-maps/audit/README.md:3]: The link is not visible in the `preview` tool, please double-check if using `>` has any impact on this.
If you find anything interesting you could update [this notion page](https://www.notion.so/01edu/Create-and-update-Audits-4b4d182743dc428b9be35f4ebf425d76).
If the problem impacts also the production code we can revert to a H4 title.

**Review by mikysett — CHANGES_REQUESTED** [review]
Except the link in the first line to double-check everything looks ok.
There is a lot of typos and small grammar error but I guess they will be assessed in the language review.

File: subjects/mobile-dev/01-maps/audit/README.md (hunk 1 above)
Comment (nprimo) [reply subjects/mobile-dev/01-maps/audit/README.md:3]: This is caused by a bug of the `preview` tool; currently the first hyperlink is skipped - to see if it properly render, try passing a line of text with a hyperlink and you should see the second one working properly

**Review by mikysett — APPROVED** [review]
LGTM

**Pull request merged**